client/branded/src/global-styles/variables.scss

```
@@ -0,0 +1,37 @@
+$font-size-base: 0.875rem;
+$line-height-base: (20/14);
+
+// Input
+$input-padding-y: 0.375rem;
+$input-padding-x: 0.75rem;
+$input-font-size-sm: (11/14) + em;
+
+// Spacer
+$spacer: 1rem;
+
+:root {
+    --spacer: #{$spacer};
+    --font-size-base: #{$font-size-base};
+    --line-height-base: #{$line-height-base};
+    --input-font-size: var(--font-size-base);
```

Let's group CSS variables by adding comments between groups like we do for SCSS variables.

```suggestion
    --line-height-base: #{$line-height-base};
    // Input
    --input-font-size: var(--font-size-base);
```